conflux/src/config.ts

```diff
@@ -0,0 +1,10 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'conflux'
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix)
+  config.api.baseURL = config.api.baseURL || 'http://localhost:12537'
+  return config
```

Comment:
You could customize your `Config` type to include some of the params needed, like the `rpcUrl`, `networkId` and `privateKey`. You can take a look in the [Defi Pulse adapter](https://github.com/smartcontractkit/external-adapters-js/blob/develop/composite/defi-pulse/src/config.ts)